app/Console/Commands/ForceDeletePosts.php

```diff
@@ -52,18 +52,21 @@ public function handle()
     {
         info('rogue:forceDeletePosts: Starting to force delete posts by type and source.');
 
-        $posts = Post::where('type', $this->argument('type'))->where('source', $this->argument('source'))->get();
+        Post::where('type', $this->argument('type'))
+            ->where('source', $this->argument('source'))
+            ->chunk(100, function ($posts) {
+                if ($posts->isNotEmpty()) {
+                    foreach ($posts as $post) {
+                        $this->postManager->destroy($post->id);
 
-        if ($posts->isNotEmpty()) {
-            foreach ($posts as $post) {
-                $this->postManager->destroy($post->id);
+                        $post->forceDelete();
+                    }
 
-                $post->forceDelete();
-            }
-
-            info('rogue:forceDeletePosts: Posts Deleted!');
-        } else {
-            info('rogue:forceDeletePosts: No Posts found');
-        }
+                    info('rogue:forceDeletePosts: Posts Deleted!');
```

Comment:
Maybe this should say "100 posts deleted" or something like that? Since this logs after a chunk is done.

Reply:
ah that is a great idea! Updated :)